### DISCUSSION TITLE
Timeline :  Human readable dates, and current time position

### DISCUSSION THREAD

**Discussion by Gogolian** [ActivityWatch/activitywatch#534] [General]
Timeline Tab:

Would you be interested to change tooltip dates to human readable format?

Also, in your opinion - should tooltips be displayed faster?

Would you be interested in timeline scroller marker (kind of vertical straight line) that follows your cursor displaying current time let's say, on bottom?

Thanks everyone and have a great day!

**Comment by NicoWeio** [comment]
I like the idea of having a scroller marker, but I think you'd need to make changes [upstream](https://github.com/visjs/vis-timeline/). 

I also like the idea of human-readable dates. In general, decluttering these tooltips is probably a good idea. Having raw JSON data and full timestamps makes them a little hard to read, especially [on mobile](https://github.com/ActivityWatch/aw-android).
